taskcluster/ci/release-notify-testrail/kind.yml

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    description: Sends Slack message to release testers that Testrail Milestone was created.
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 120
+        env:
+            TESTRAIL_PROJECT_ID: '59' # Fenix Browser
+            TESTRAIL_TEST_SUITE_ID: '49319' # Test Automation Release Milestone
+    run:
+        use-caches: false
+        using: run-commands
+        secrets:
+            - name: project/mobile/ci/testrail
```

Comment:
not `project/mobile/firefox-android/testrail`?

Reply:
Since we'll be using those credentials for both `firefox-android` and `firefox-ios`, I think ahal recommended using that naming convention, but it was maybe a year ago, it feels like, when we talked about it.

I'm not sure what the best practice is here, and am open to any recommendations on `best-practices` for organizing and managing the `Taskcluster` assets/dependencies like this.

Reply:
I see: that makes sense then. Let's keep it like this.